### PR TITLE
Publish `.deb` package file in release

### PR DESCRIPTION
**Pull request by devtomio**
Publishing a `.deb` file in release allows Nixpacks to be installed with `dpkg`.

- [x] Docs are updated if needed

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -137,6 +137,20 @@ jobs:
...
+    - name: Install cargo-deb
+      if: matrix.target == 'x86_64-unknown-linux-musl'
+      run: cargo install cargo-deb
+
+    - name: Generate .deb package file
+      if: matrix.target == 'x86_64-unknown-linux-musl'
+      run: cargo deb --target x86_64-unknown-linux-musl --output nixpacks-${{ needs.create-release.outputs.nixpacks_version }}-amd64.deb
+
+    - name: Upload .deb package file
+      uses: svenstaro/upload-release-action@v2
```
Comment (nebulatgs) [inline .github/workflows/release.yml:149]:
```suggestion
      if: matrix.target == 'x86_64-unknown-linux-musl'
      uses: svenstaro/upload-release-action@v2
```
dont we need to check here as well?

Comment (nebulatgs) [inline .github/workflows/release.yml:149]:
```suggestion
      if: matrix.target == 'x86_64-unknown-linux-musl'
      uses: svenstaro/upload-release-action@v2
```
dont we need to check here as well?

**Review by nebulatgs — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (nebulatgs) [reply .github/workflows/release.yml:149]: since otherwise the `.deb` file won't exist

Comment (devtomio) [reply .github/workflows/release.yml:149]: oops, thank you

**Review by coffee-cup — APPROVED** [review]

**Pull request merged**